proposals/0000-unsaturated-type-families.rst

```diff
@@ -298,8 +403,20 @@ the same arrow ``->``. Consequently, as a result of this proposal, term-level
 functions inevitably need to have a matchability. They are assigned
 the unmatchable arrow ``-> @U``.
 
-Inferred arguments
-~~~~~~~~~~~~~~~~~~
+Visible dependent quantification
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Another place where the ``->`` shows up today is visible dependent
+quantification, for example in the kind of ::
+
+  -- P :: forall k -> k -> Type
+  data P k (a :: k) = P
+
+We propose to annotate such arrows with matchabilities too, so the full
+kind of ``P`` becomes ``forall k -> @M -> k -> @M Type``.
```

Comment:
Did you mean `forall k -> @M k -> ...`? I think there's an extra arrow here.

Reply:
Indeed, thanks!